koch.nim

```diff
@@ -465,7 +465,7 @@ proc temp(args: string) =
   if "doc" notin programArgs and
       "threads" notin programArgs and
       "js" notin programArgs and "rst2html" notin programArgs:
-    bootArgs.add " -d:leanCompiler"
+    bootArgs = " -d:leanCompiler" & bootArgs
```

Comment:
seems correct but only because of undocumented fact that splitArgs[0] prepends `" "` before each argument; we should either document this or add a harmless `" "`:
`bootArgs = " -d:leanCompiler " & bootArgs`